src/concepts/plugins/frontend_sdk.md

```diff
@@ -0,0 +1,201 @@
+# Frontend Plugin SDK
```

Comment:
Its kinda to always keep the documentation up to date with the typing so in general I really prefer we give usage examples instead which are less likely to change between versions instead of a dump of typescript typing info.

Reply:
I am still very much on the fence about this documentation. Lets put it as WORK IN PROGRESS for now and not include the full typing

Reply:
Want me to just remove the page entirely? This is a portion of this PR I haven't worked on since creation.

Reply:
Done.